src/ecalc/libraries/libecalc/common/libecalc/input/yaml_types/components/pump_system.py

```diff
@@ -204,7 +205,8 @@ def to_dto(
                 else:
                     rates = [
                         Expression.multiply(
-                            Expression.setup_from_expression(self.rate), Expression.setup_from_expression(rate_fraction)
+                            Expression.setup_from_expression(operational_setting.total_system_rate),
```

Comment:
oops, forgot to mention I fixed a bug

Reply:
nice to catch this..another "pros" of having a type/alias rather than typing the Union etc all the time...

Reply:
Ideally separate commit though...